zboxcore/sdk/allocation.go

```diff
@@ -788,6 +803,69 @@ func (a *Allocation) getRefs(path, pathHash, authToken, offsetPath, updatedDate,
 	return oTreeReq.GetRefs()
 }
 
+func (a *Allocation) getDownloadMaskForBlobber(blobberID string) (zboxutil.Uint128, []*blockchain.StorageNode, error) {
+
+	x := zboxutil.NewUint128(1).Lsh(uint64(1)).Sub64(1)
```

Comment:
The mask is always `{0, 1}` no matter what the blobberID is, is this correct?

Reply:
yes this value will always be `{0, 1}`. hence, to simplify, made it `x := zboxutil.NewUint128(1)`